common/src/events/events.types.ts

```diff
@@ -16,9 +17,11 @@ export enum EventTopic {
 
     // Submissions
     SubmissionStarted = "submission.started",
-}
-
 
+    // Member
+    AllMemberEvents = "member.*",
+    MemberBanned = "member.banned",
```

Comment:
Can we put this as `member.restriction.created`? Or `member.queue-banned` or `member.restriction.queue-banned`? `banned` itself is too vague.

Reply:
`member.restriction.created` is probably the most correct way to do this, filtering should happen on the receiving end.

Reply:
Works for me.